README.md

```diff
@@ -159,6 +159,63 @@ You'll need to upload these to an s3 bucket, then you can use these files to
 [create a simulation application](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-application.html),
 and [create a simulation job](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-job.html) in RoboMaker.
 
+
+## Generate map for your world
+
+Requires ROS-Melodic + Gazebo9.
```

Comment:
Can you add a blurb about what "Generate a map for your world" is for? And is it needed if the user is using the bookstore world?

Reply:
Done.